src/js/components/DataTable/DateCell.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+
+import moment from 'moment';
+import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import TableCell from 'components/DataTable/TableCell';
+
+const DateCell = ({ tableDateFormat, ...row }) => (
+  <TableCell
+    {...row}
+    value={row.value && moment(row.value).format(tableDateFormat)}
+    defaultValue="-"
```

Comment:
shouldn't **defaultValue** also be customizable instead of hardcoding a **"-"**?

Reply:
it could be, why not, I didn't pay that much attention to `defaultValue` since it was only copied from the already existing code, but makes sense